sdk/src/types/block/core.rs

```diff
@@ -321,6 +321,7 @@ impl Block {
 
     /// Creates a new [`BlockBuilder`] to construct an instance of a [`ValidationBlock`].
     #[inline(always)]
+    #[allow(clippy::too_many_arguments)]
```

Comment:
How did this pass CI in a previous PR ?

Reply:
We disabled the deny for warnings

Reply:
Temporarily?

Reply:
Yes, until we iron out some design around issuing blocks.